src/main/scala/is/hail/methods/LDMatrix.scala

```diff
@@ -40,12 +41,30 @@ object LDMatrix {
     normalizedBlockMatrix.blocks.count()
     normalizedIRM.rows.unpersist()
 
-    val indexedRowMatrix = (normalizedBlockMatrix multiply normalizedBlockMatrix.transpose)
-      .toIndexedRowMatrix()
+    val thresholdSize = 5000 * 5000
+    val numEntries: Long = normalizedBlockMatrix.numCols() * normalizedBlockMatrix.numRows()
```

Comment:
Yeah it would be expensive. That needlessly would do a count. I should change `toBlockMatrixDense` to include the count in constructor for BlockMatrix so this is a constant time operation in the future. Will make an issue, but will also just use `variantsKept.length` here